lib/adal.js

```diff
@@ -169,8 +163,7 @@ var AuthenticationContext = (function () {
     };
 
     /**
-     * Gets initial Idtoken for the app backend
-     * Saves the resulting Idtoken in localStorage.
+     * Gets initial Idtoken for the app's backend and saves it in the cache.
```

Comment:
We should update this to something like: "Initiates the login process, redirecting or prompting the user to Azure AD login page".